features/netsocket/WiFiInterface.h

```diff
@@ -89,13 +89,15 @@ class WiFiInterface: public virtual NetworkInterface {
 
     /** Scan for available networks.
      *
-     *  This function will block. If the count is 0, function will only return count of available networks, so that
-     *  user can allocated necessary memory. If the count is grater than 0 and the a \p res is not NULL it'll be populated
-     *  with discovered networks up to value of count.
+     *  @note This function will block.
```

Comment:
Function blocks what?

Reply:
It is a blocking function, it prevents execution (it waits for something to happen) on this thread and possibly on the driver and the device this call goes to, until the function has something to return. In this case, until the Wi-Fi scan has been completed.

Reply:
I think "this is a blocking function" is clearer than "this function will block".

Reply:
Agreed, will change

Reply:
Done